traits.yml

```diff
@@ -73,6 +73,43 @@ traits:
 
               This must be a valid URL so special characters need to
               be encoded.
+  identity:
```

Comment:
[question] I wonder if `LocateableContent` is also a type of "identity"?

Reply:
It felt different to me, as were not a content-addressed system (by design), so having the "data" for something be part of its identity might muddy the waters some?